jacobian_follower/test/jacobian_follower_bindings_test.py

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env python3
+# Note: python3 needs to be hardcoded for rostests to work properly: https://github.com/ros/ros_comm/issues/1830
+
+from arm_robots.victor import Victor
+import unittest
+import rostest
+import rospy
+from geometry_msgs.msg import Pose
+
+
+
+
+class TestJacobianFollowerBindings(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.victor = Victor()
+        self.victor.connect()
```

Comment:
man I wish we didn't need this. shouldn't need anything other than the description + moveit config params for these test to work :frowning:

Reply:
Agreed. Downsides of connect are
- Need a `Victor` to connect to (so far I've already had a `Victor` running when I need a jacobian follower)
- `connect` takes a few seconds